### ISSUE TITLE
Tabs for hardware platform on getting started page

### ISSUE THREAD

**Issue by laurensvalk** [ev3dev/ev3dev.github.io#230]
The getting started page is getting somewhat cluttered with buttons and exceptions for the various hardware platforms. 

What about using tabs for EV3/Raspberry Pi/Beablegone? (Like you do for Mac/Ubuntu/Windows on the networking page).

It adds some slight overhead for the maintainers, but it's only the getting started page.

**Comment by dlech** [comment]
Sounds like ev3dev/ev3dev#474

**Comment by WasabiFan** [comment]
Yeah, this is one of the things I expect for the issue that @dlech linked to (as well as #87) to help solve. That work is currently solved pending a design decision; I've intended to make UI mockups so I could start approaching a decision on how it should be build, but I've been preoccupied so far.
